index.html

```diff
@@ -29,7 +34,7 @@
     <section class="header-page">  
```

Comment:
The text of the headline section and social icons should be at the center of the page. Moreover, please add more space between social icons. KIndly see the screenshot below
![headline](https://user-images.githubusercontent.com/35504149/135630557-4d8d4b63-e66e-48b5-9782-d4dcbc9450c7.png)
.